x-pack/docs/en/rest-api/security/get-user-profile.asciidoc

```diff
@@ -1,10 +1,134 @@
 [role="xpack"]
 [[security-api-get-user-profile]]
 === Get user profile API
+
+beta::[]
+
 ++++
 <titleabbrev>Get user profile</titleabbrev>
 ++++
 
-Retrieves a user's profile given the unique profile ID
+Retrieves a user's profile using the unique profile ID.
+
+[[security-api-get-user-profile-request]]
+==== {api-request-title}
+
+`GET /_security/profile/<uid>`
+
+[[security-api-get-user-profile-prereqs]]
+==== {api-prereq-title}
+
+* To use this API, you must have _at least_ the `manage_user_profile` cluster privilege.
+
+
+[[security-api-get-user-profile-desc]]
+==== {api-description-title}
+
+The get user profile API returns the user profile document that matches the specified `uid`.
```

Comment:
@ywangd, how do you obtain the `uid` to pass it into the get user profile request? The only place that I've found it is in the response from the activate user profile API. Is there another way to obtain the `uid`, or is it only available when activating the user profile?

Reply:
Yes that's the only place where you can get the `uid`. The intended audience of these APIs is Kibana (or applications running on top of Kibana) and the envsioned workflow is that Kibana will call ActivateProfile on user login and keep the reference of its `uid` throughout the session.

Reply:
Thanks @ywangd -- I clarified this sentence in f45ad57d1d7188692e4ae7cf9ab80ee639318d20 and included a link to the activate user profile API page:

>The get user profile API returns the user profile document matching a specified 
`uid`, which is generated when
<<security-api-activiate-user-profile,activating a user profile>>.